notifier/helper.py

```diff
@@ -11,7 +11,9 @@ def get_project_id(request_id):
 def generate_sample_data_content(files, pipeline_name, pipeline_github, pipeline_version):
     result = "SAMPLE_ID\tREQUEST_ID\tPROJECT_ID\tPATIENT_ID\tCOLLAB_ID\tSAMPLE_TYPE\tGENE_PANEL\tONCOTREE_CODE\tSAMPLE_CLASS\tSPECIMEN_PRESERVATION_TYPE\tSEX\tTISSUE_SITE\tIGO_ID\tPIPELINE_NAME\tPIPELINE_GITHUB_LINK\tPIPELINE_VERSION\n"
     ret_str = 'metadata__sampleId'
-    query = Q(file__path__in=files) & Q(file__file_group_id=settings.IMPORT_FILE_GROUP)
+    query = Q(file__file_group_id=settings.IMPORT_FILE_GROUP)
+    query |= Q(file__file_group__slug="origin-unknown")
```

Comment:
This assumes the file, if found, are structured the same way as in the LIMS filegroup.

This is needed because I suspect non-LIMS data sources will go through Helix Filters, requiring data clinical files.